crates/ui/src/windows/items.rs

```diff
@@ -69,66 +65,454 @@ impl luminol_core::Window for Window {
+                                                            id,
+                                                            format!("{:0>3}: {}", id, item.name),
+                                                        );
+                                                    });
+                                                }
+                                            },
+                                        );
+
+                                    if ui
+                                        .add(egui::Button::new(change_maximum_text).wrap(false))
+                                        .clicked()
+                                    {
+                                        luminol_core::basic!(
+                                            update_state.toasts,
+                                            "`Change maximum...` button trigger"
+                                        );
+                                    }
+                                },
+                            );
+                        });
                 });
+
+                egui::Frame::none()
+                    .outer_margin(egui::Margin {
+                        left: ui.spacing().window_margin.left,
+                        ..egui::Margin::ZERO
+                    })
+                    .show(ui, |ui| {
+                        ui.with_layout(
+                            egui::Layout {
+                                cross_justify: true,
+                                ..Default::default()
+                            },
+                            |ui| {
+                                egui::ScrollArea::vertical().show(ui, |ui| {
+                                    ui.set_width(ui.available_width());
+
+                                    let selected_item = &mut items.data[self.selected_item];
+
+                                    modified |= ui
+                                        .add(luminol_components::Field::new(
+                                            "Name",
+                                            egui::TextEdit::singleline(&mut selected_item.name)
+                                                .desired_width(f32::INFINITY),
+                                        ))
+                                        .changed();
+
+                                    modified |= ui
+                                        .add(luminol_components::Field::new(
+                                            "Description",
+                                            egui::TextEdit::multiline(
+                                                &mut selected_item.description,
+                                            )
+                                            .desired_width(f32::INFINITY),
+                                        ))
+                                        .changed();
+
+                                    egui::Frame::none().fill(ui.visuals().faint_bg_color).show(
```

Comment:
egui_extra's table might be better suited for this?

Reply:
That requires determining the height of each row beforehand. I don't think that's practical since the combo box labels can wrap if text is too long.

Reply:
I can't get the combo boxes to not wrap, either. The API only allows for wrapping combo boxes or combo boxes that don't wrap but automatically expand the parent widget to fit. There's no easy way to get it to truncate the width of the combo box to fit into the parent widget without messing up something else, as far as I can tell.